analyzers/src/SonarAnalyzer.Common/Helpers/MethodParameterLookupBase.cs

```diff
@@ -69,12 +70,22 @@ private bool TryGetSymbol(SyntaxNode argument, IMethodSymbol methodSymbol, out I
             return false;
         }
 
-        if (GetNameColonArgumentIdentifier(arg) is { } nameColonArgumentIdentifier)
+        if (GetNameColonIdentifier(arg) is { } nameColonArgumentIdentifier)
         {
             parameter = methodSymbol.Parameters.FirstOrDefault(symbol => symbol.Name == nameColonArgumentIdentifier.ValueText);
             return parameter != null;
         }
 
+        if (GetNameEqualsIdentifier(arg) is { } nameEqualsArgumentIdentifier
+            && methodSymbol.ContainingType.GetMembers(nameEqualsArgumentIdentifier.ValueText) is { Length: 1 } properties
```

Comment:
Doesn't this mean that we will not support the property set, if there is a method with the same name as the property?

Reply:
How can you define a method with the same name as a property?